app/build.gradle

```diff
@@ -11,12 +11,13 @@ android {
         targetSdkVersion rootProject.targetSdkVersion
         versionCode rootProject.versionCode
         versionName rootProject.versionName
+        multiDexEnabled true
```

Comment:
Do we need this?

Reply:
Yeah, you cannot launch sample app without it :(